controllers/foodsController.js

```diff
@@ -0,0 +1,50 @@
+const Food = require('../models/food')
+
+const index = (req, res, next) => {
+  Food.findAll()
+    .then(foods => {
+      if(!foods) {
+        return res.sendStatus(404)
+      } else {
+        res.json(foods)
+      }
+  })
+}
+
+const show = (req, res, next) => {
+  let id = req.params.id
+
+  Food.find(id)
+    .then(food => {
+      if(!food) {
```

Comment:
This error handling looks the same as above, so that's a possible candidate for refactoring.